### DISCUSSION TITLE
Is there any documentation or advice on how to style fluent v9 components?

### DISCUSSION THREAD

**Discussion by daric81** [microsoft/fluentui#25751] [Q&A] [answered]
My team are migrating from v8 > v9 and are struggling with styling components. I cant seem to find documentation regarding this. v8 documentation around component style props seemed more complete, unless I'm missing something fundamental.

**Comment by AkiraVoid** [comment, marked as answer]
Take a look: https://react.fluentui.dev/?path=/docs/concepts-developer-styling-components--page and https://react.fluentui.dev/?path=/docs/concepts-developer-customizing-components-with-slots--page

And you may need [griffel docs](https://griffel.js.org).

By the way, all design tokens (colors, margins, speed curves, animations, etc.) are exported as `tokens` object in Fluent UI v9 (see https://react.fluentui.dev/?path=/docs/concepts-developer-theming--page).
